ppd/testppdfile.c

```diff
@@ -16,6 +16,26 @@
 #include <ppd/array-private.h>
 #include <stdio.h>
 
+
+/*
+ * Error warning overrides...
+ */
+
+enum
+{
+  WARN_NONE = 0,
+  WARN_CONSTRAINTS = 1,
+  WARN_DEFAULTS = 2,
+  WARN_FILTERS = 4,
+  WARN_PROFILES = 8,
+  WARN_TRANSLATIONS = 16,
+  WARN_DUPLEX = 32,
+  WARN_SIZES = 64,
+  WARN_FILENAME = 128,
+  WARN_ALL = 255
+};
+
+
```

Comment:
Please move these enums into ppd/ppd.h, as everyone who wants to call `ppdTest()` needs them. Then `ppdTest()` itself can use them, `testppfile.c`can use them and everyone who does `#include <ppd/ppd.h>` for using `ppdTest()` ...